### ISSUE TITLE
Design responses for Lock Ratio interaction

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#221]
@terracoda is to work on this

**Comment by terracoda** [comment]
@zepumph, I created a table here in the design doc. I only have No Tick Marks column filled it, but I have flipped thing around.

Will something like be possible?
- Object value = proximity to ratio
- Context response = something useful about distance or distance progress and Play Area position for hands.

I was also playing with the logic for when someone gets a "distance progress" response.
1. On first step or change in direction
2. The first step after a distance region change
3. When there's a Play area position region change for either hand or both hands

We may need a special response (a go beyond edge response) if one hand is at the top or bottom and the user keeps trying to move the hands.

[Link to Table is here](https://docs.google.com/document/d/1HJvvGWYaUv9tlat9S7omQQVng8fGB4FuUhTEM1qx1z8/edit#heading=h.tdb6deghc0kc)

I still need to see how this would work for the tick marks and numbered tick marks. The first column would be the same for all.

**Comment by terracoda** [comment]
@zepumph, could you answer if this approach is possible/doable and re-assign to me to fill in the other tick marks?

**Comment by zepumph** [comment]
This seems like it would be possible from my view.

**Comment by zepumph** [comment]
@terracoda has completed the first column in the above table (for qualitative description), and on slack we decided to implement what is in there as best as we can for now, and discuss it tomorrow.

**Comment by zepumph** [comment]
* Bug fix for when distance was calculated after both values had changed (like when ratio is locked).
* Update single hand interaction context response when locked to be something like "Hands, very close to each other, both hands in lower region."
* Update single hand interaction object response when locked to be the proximity to ratio, which means that it is always "at challenge ratio"
* Noted that currently, at least for NVDA, AccessibleValueHandler isn't repeating duplicate aria-valuetext, so I created https://github.com/phetsims/scenery-phet/issues/640.

I'm not very confident about the above changes. There were a few too many changes that I didn't expect to encounter. That is just to make you aware of the current state of this part of description, so keep your eyes peeled with me.

This is at least ready for review and discussion in tomorrows meeting. Let me know what you think @terracoda!


https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-dev.70/phet/ratio-and-proportion_en_phet.html

**Comment by zepumph** [comment]
This feature was added to the "both hands" interaction when the ratio is locked. See above. Up next is the "Hands, hands. . ." typo.

**Comment by zepumph** [comment]
>Noted that currently, at least for NVDA, AccessibleValueHandler isn't repeating duplicate aria-valuetext, so I created phetsims/scenery-phet#640.

This is fixed now.

**Comment by zepumph** [comment]
I also saw a recent change in the design doc, and now the object description is "hands, at challenge ratio".

**Comment by zepumph** [comment]
Ready for review.

**Comment by terracoda** [comment]
@zepumph, this sounds good on my system.
Nice job. I am getting the object response "hands, at challenge ratio" for both sliders and teh Both Hands interaction.

I am getting good context responses, too.

**Comment by terracoda** [comment]
There's an issue with the sound on my system. I'll make a separate issue for that to address after this weeks interviews.

**Comment by terracoda** [comment]
The lock ratio responses were implemented as I requested, so closing this issue.
